packages/core/src/messages/responses/storage/control/list-stores.ts

```diff
@@ -0,0 +1,54 @@
+import {ListStoresResponse} from '../../enums';
+import {BaseResponseError, BaseResponseSuccess} from '../../response-base';
+import {StoreInfo} from '../../../store-info';
+import {SdkError} from '../../../../errors';
+
+interface IResponse {
+  readonly type: ListStoresResponse;
+}
+
+/**
+ * Indicates a successful list stores request.
+ */
+export class Success extends BaseResponseSuccess implements IResponse {
+  readonly type: ListStoresResponse.Success = ListStoresResponse.Success;
+  private readonly stores: StoreInfo[];
+
+  constructor(stores: StoreInfo[]) {
+    super();
+    this.stores = stores;
+  }
+
+  /**
+   * An array of StoreInfo, containing information about each store.
+   * @returns {StoreInfo[]}
+   */
+  public getStores() {
+    return this.stores;
+  }
```

Comment:
```suggestion
  public stores() {
    return this._stores;
  }
```
nit: I think to be consistent with the cache response types, don't include `get` in the name